wordle-web/pages/game.vue

```diff
@@ -1,105 +1,215 @@
+      :type="gameResult.type"
+    >
       <v-row justify="center">
-        <game-board :wordleGame="wordleGame" />
-      </v-row>
-      <v-row justify="center">
-        <keyboard :wordleGame="wordleGame" />
+        <v-col>
+          <p>{{ gameResult.text }} <br /></p>
+        </v-col>
       </v-row>
-    </v-container>
+      <v-row>
+        <v-btn class="ml-2" :disabled="wordleGame.hasLost" @click="submitScore">
+          Submit Score
+        </v-btn>
+        <v-btn class="ml-2" @click="resetGame"> Play Again </v-btn></v-row
+      >
+    </v-alert>
+
+    <v-row justify="center"
+      ><v-banner v-model="submittedSuccess" color="success" rounded single-line>
+        Your Score has been submitted!
+      </v-banner></v-row
+    >
+
+    <v-row justify="center">
+      <game-board :wordleGame="wordleGame" />
+    </v-row>
+    <v-row justify="center">
+      <keyboard :wordleGame="wordleGame" />
+    </v-row>
   </v-container>
 </template>
 
 <script lang="ts">
-import { Component, Vue } from 'vue-property-decorator'
+import { Component, Vue, Watch } from 'vue-property-decorator'
 import { WordsService } from '~/scripts/wordsService'
 import { GameState, WordleGame } from '~/scripts/wordleGame'
 import KeyBoard from '@/components/keyboard.vue'
 import GameBoard from '@/components/game-board.vue'
 import { Word } from '~/scripts/word'
+import { Stopwatch } from 'ts-stopwatch'
 
 @Component({ components: { KeyBoard, GameBoard } })
 export default class Game extends Vue {
   word: string = WordsService.getRandomWord()
   wordleGame = new WordleGame(this.word)
-
-  isLoaded: boolean = false
+  username: string = this.evaluateName()
+  dialog: boolean = false
+  submittedSuccess: boolean = false
+  gameTimeSeconds: number = 0
+  stopwatch = new Stopwatch()
+  rules = {
+    required: (value: any) => !!value || 'Required.',
+    counter: (value: string | any[]) =>
+      value.length <= 25 || 'Max 25 characters',
+  }
 
   mounted() {
-    setTimeout(() => {
-      this.isLoaded = true
-    }, 5000)
+    this.startTime()
+  }
+
+  startTime() {
+    this.stopwatch.start()
+  }
+
+  stopTime() {
+    this.stopwatch.stop()
+  }
+
+  getTime() {
+    return this.stopwatch.getTime()
+  }
+
+  resetTime() {
+    this.stopwatch.reset()
+  }
+
+  getTimeSeconds() {
+    return this.getTime() / 1000
   }
 
   resetGame() {
+    this.submittedSuccess = false
     this.word = WordsService.getRandomWord()
     this.wordleGame = new WordleGame(this.word)
+    this.resetTime()
+    this.startTime()
+  }
+
+  submitScore() {
+    //guard statement, only submit score once
+    if (this.submittedSuccess) return
+
+    if (!this.hasUniqueLocalName()) {
+      this.toggleDialog()
+      this.waitUntilDialogClosed()
+    } else {
+      this.$axios.post('/api/PlayersLeaderBoard', {
+        name: this.username,
+        attempts: this.wordleGame.currentGuess,
+        seconds: this.getTimeSeconds().toFixed(0), //removes after numbers decimal
```

Comment:
An interesting choice to remove the decimal point entirely, I would suggest either leaving it as you have it on the success message when it reports the time to the player after they won, or truncating that one as well.